core/arch/arm/mm/tee_pager.c

```diff
@@ -532,9 +532,14 @@ void tee_pager_add_core_area(vaddr_t base, enum tee_pager_area_type type,
 	uint32_t flags = 0;
 	size_t fobj_pgoffs = 0;
 	vaddr_t b = base;
-	size_t s = fobj->num_pages * SMALL_PAGE_SIZE;
+	size_t s;
 	size_t s2 = 0;
 
+	if (!fobj)
+		panic();
```

Comment:
I would prefer an assertion here despite the fact there is an explicit (but useless) `panic()` in current implementation.

Reply:
@etienne-lms, no strong opinion here, but I agree that you have a point.

Reply:
added a string to the panic (it was a paninc withotu a string too late in teh function before)

ok as is?